src/System.Net.Mail/tests/Unit/EncodedStreamFactoryTest.cs

```diff
@@ -2,47 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.IO;
 using System.Text;
 using Xunit;
 
 namespace System.Net.Mime.Tests
 {
     public class EncodedStreamFactoryTests
     {
-        [Fact]
```

Comment:
Why are you deleting tests?  Tests should not be deleted.

Reply:
These tests were removed because the relevant methods were removed [here](https://github.com/dotnet/corefx/pull/29302/files#diff-f106cd330f5aea61da67184a76a26d0cL21)
and [here](https://github.com/dotnet/corefx/pull/29302/files#diff-d3c7d3b0ebd5e40867f4e209450c47d7L14). These were removed in accordance with #17905. (See http://tempcoverage.blob.core.windows.net/report3/System.Net.Mail.diff.html)